results.py

```diff
@@ -188,10 +195,10 @@ def to_png(self, view_name: str, process_name: str, file_name: str):
             y="Query Ratio",
             size="Mean Process Time" if process_name is not None else None,
             hue="Mode",
-            palette="viridis",
+            palette=PALETTE,
             edgecolors="black",
             legend=True,
-            alpha=0.4,
+            alpha=0.8,
```

Comment:
with different colours, the brighter dots work better

Reply:
Nice one Thanks